packages/mdc-drawer/temporary/mdc-temporary-drawer.scss

```diff
@@ -45,15 +46,16 @@
     box-sizing: border-box;
     width: 100%;
     height: 100%;
-    background: rgba(0, 0, 0, .6);
+    @include mdc-drawer-scrim-color($mdc-temporary-drawer-scrim-color, $mdc-temporary-drawer-scrim-opacity);
     opacity: 0;
     opacity: var(--mdc-temporary-drawer-opacity, 0);
     content: "";
     will-change: opacity;
   }
 
   &__drawer {
-    @include mdc-theme-prop(background, background);
+    @include mdc-drawer-ink-color($mdc-temporary-drawer-ink-color);
```

Comment:
We've generally been `@include`-ing the theme mixins on the root element rather than child elements, and then putting the child element selectors in the mixins.

That approach allows clients to keep all (or most) of their theme customizations in one place.

Reply:
fixed